autoload/airline/section.vim

```diff
@@ -13,6 +12,7 @@ function! s:wrap_accent(part, value)
 endfunction
 
 function! s:create(parts, append)
+  let s:spc = g:airline_symbols.pad_sep
```

Comment:
I kept the same name so as to minimize the pull request.

I will make these changes if you are interested in pulling this